cpp/tests/CMakeLists.txt

```
@@ -80,6 +80,7 @@ include_directories("${CMAKE_BINARY_DIR}/include"
                     "${CMAKE_SOURCE_DIR}/thirdparty/cub"
                     "${CMAKE_SOURCE_DIR}/thirdparty/jitify"
                     "${CMAKE_SOURCE_DIR}/thirdparty/dlpack/include"
+                    "${DLPACK_ROOT}/include"
```

We previously defined this in the CMake, would also suggest moving this to below `RMM_INCLUDE` to be consistent with elsewhere.
```suggestion
                    "${DLPACK_INCLUDE}"
```